src/Maestro/maestro-angular/src/app/model/version-details.ts

```diff
@@ -0,0 +1,279 @@
+import { Subscription, Asset, Build } from 'src/maestro-client/models';
+import { Observable, combineLatest } from 'rxjs';
+import { map, switchMap } from 'rxjs/operators';
+import { StatefulResult, statefulPipe } from 'src/stateful';
+import { BuildService } from 'src/app/services/build.service';
+import { WrappedError, Loading } from 'src/stateful/helpers';
+import { DependencyDetail } from './dependency-detail';
+
+export class VersionDetails {
```

Comment:
This feels like the javascript is taking an really hard dependency on the mechanism by which we deal with dependencies.  Couldn't we have the server manage this and return some version information in an API?  Parsing arbitrary XML with special strings all over is going to be difficult to maintain.